src/containers/lists/BidList/BidList.tsx

```diff
@@ -43,17 +47,22 @@ const BidList: React.FC<BidListProps> = ({
 	isAccepting,
 	isLoading,
 	highestBid,
+	isAcceptBidEnabled = false,
 }) => {
 	//////////////////
 	// Data & State //
 	//////////////////
 	const [blockNumber, setBlockNumber] = useState<number>();
 	const [isAcceptBidModal, setIsAcceptBidModal] = useState(false);
 	const [acceptingBid, setAcceptingBid] = useState<Bid | undefined>(undefined);
-	const { library } = useWeb3React<Web3Provider>();
+	const { library, account } = useWeb3React<Web3Provider>();
 
 	// Sort bids by date
 	const sortedBids = sortBidsByTime(bids);
+	const filteredBids = sortedBids.filter(
+		(bid) => bid.bidder.toLowerCase() !== account?.toLowerCase(),
+	);
+	const isFilteredBids = isAcceptBidEnabled ? filteredBids : sortedBids;
```

Comment:
tiny nit: `isFilteredBids` implies the variable is a boolean (because of `is`) - I think it would be clearer to call it `filteredBid`, `bidsToShow`, or something similar.